logic/peers.js

```diff
@@ -254,6 +254,24 @@ Peers.prototype.list = function(normalize) {
 	);
 };
 
+/**
+ * Returns a random list of connected peers.
+ *
+ * @param {Object} [options] - Optional
+ * @param {number} [options.limit=null] - Maximum number of peers to get; defaults to all
+ * @returns {peer[]} List of peers
+ */
+Peers.prototype.listRandomConnected = function(options) {
```

Comment:
Why that function is in `logic/peers` and not in `modules/peers` as `list`?

Reply:
My thinking was that ideally modules shouldn't call methods of each other (since they already communicate via message passing) and that logic/ classes exist for the purpose of sharing overlapping functionality between modules.